mkdocs.yml

```diff
@@ -1,7 +1,8 @@
 site_name: JDA Wiki
-copyright: Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß
+copyright: 'Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß and <a href="https://github.com/DV8FromTheWorld/JDA-Website/graphs/contributors" target="_blank">Wiki Contributors</a>'
```

Comment:
```suggestion
copyright: 'Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and <a href="https://github.com/DV8FromTheWorld/JDA-Website/graphs/contributors" target="_blank">Wiki Contributors</a>'
```

Reply:
I feel like it would be somewhat better to just change it to `Copyright (c) JDA Wiki Contributers`?

Reply:
I'd prefer to keep what we have now, for example michael doesn't get listed in the contributors and the content attribution (git blame) is lost due to the way we setup the repo.